client/src/app/core/core-services/operator.service.ts

```diff
@@ -345,10 +349,15 @@ export class OperatorService {
      * This behaviour prevents a non-redirect on the startpage.
      */
     private navigateOnLogout(): void {
+        // First, check if a user is at the dashboard
```

Comment:
not only the dashboard, but also the committee list etc. Just make this depending on the active meeting id - if there is none, redirect to `/login`, else to the meeting specific one. With the recent changes in #209 the meeting id should be correct